src/hotel_reservations/data_processor.py

```
@@ -1,50 +1,129 @@
+
+        return train_set_spark, test_set_spark
+
+    def create_feature_table(self, spark: SparkSession):
+        spark.sql(f"""
+        CREATE OR REPLACE TABLE {self.fe_table_name}
+        (Booking_ID STRING NOT NULL,
+        type_of_meal_plan STRING);
+        """)
+
+        spark.sql(f"ALTER TABLE {self.fe_table_name} " "ADD CONSTRAINT hotel_reservations_pk PRIMARY KEY(Booking_ID);")
+        spark.sql(f"ALTER TABLE {self.fe_table_name} " "SET TBLPROPERTIES (delta.enableChangeDataFeed = true);")                
```

Ruff

Trailing whitespace

Remove trailing whitespace

(W291)